alert/blueprints/application/routes.py

```diff
@@ -11,9 +13,18 @@
 def home():
     form = EditGrammarForm()
     if form.validate_on_submit():
-        graph_script.graph_of_grammar(form.grammar_editor.data)
+        filename = random.randint(1, 10000000)
+        graph_script.graph_of_grammar(form.grammar_editor.data, filename=str(filename))
+        return render_template(
+            "application/home.html", form=form, filename=str(filename)
+        )
+
+    return render_template("application/home.html", form=form, filename="frog")
+
 
-    return render_template("application/home.html", form=form)
+# @application.route("/gen_graph", methods=['POST'])
```

Comment:
There shouldn't be any commented out code in the source control so you can delete this.